recOrder/calib/Calibration.py

```diff
@@ -767,63 +767,55 @@ def calc_inst_matrix(self):
+                        'Swing (waves)': self.swing,
+                        'Wavelength (nm)': self.wavelength,
+                        'Black level': np.round(self.I_Black, 2),
+                        'Extinction ratio': self.extinction_ratio,
+                        'ROI (x, y, width, height)': self.ROI},
                     'Notes': notes,
-                    'Microscope Parameters': microscope_params
+                    'Microscope parameters': microscope_params
                     }
 
+        if self.calib_scheme == '4-State':
+            metadata['Calibration'].update({
+                'Channel names': [f"State{i}" for i in range(4)],
+                'LC retardance': {f'LC{i}_{j}': np.around(getattr(self, f'lc{i.lower()}_{j}'), decimals=5)
+                                  for j in ['ext', '0', '60', '120']
+                                  for i in ['A', 'B']},
+                'LC voltage': {},
+                'Swing_0': np.around(self.swing0, decimals=3),
+                'Swing_60': np.around(self.swing60, decimals=3),
+                'Swing_120': np.around(self.swing120, decimals=3),
+                'Instrument matrix': inst_mat
+            })
+
         elif self.calib_scheme == '5-State':
-            data = {'Summary':
-                    {'Timestamp': str(datetime.now()),
-                     'recOrder-napari version': str(pkg_resources.require("recOrder-napari")[0]),
-                     'waveorder version': str(pkg_resources.require("waveorder")[0]),
-                     'Acquired Using': '5-State',
-                     'Swing (fraction)': self.swing,
-                     'Wavelength (nm)': self.wavelength,
-                     'BlackLevel': np.round(self.I_Black, 2),
-                     'Extinction Ratio': self.extinction_ratio,
-                     'ChNames': ["State0", "State1", "State2", "State3", "State4"],
-                     '[LCA_Ext, LCB_Ext]': [self.lca_ext, self.lcb_ext],
-                     '[LCA_0, LCB_0]': [self.lca_0, self.lcb_0],
-                     '[LCA_45, LCB_45]': [self.lca_45, self.lcb_45],
-                     '[LCA_90, LCB_90]': [self.lca_90, self.lcb_90],
-                     '[LCA_135, LCB_135]': [self.lca_135, self.lcb_135],
-                     'Swing0': self.swing0,
-                     'Swing45': self.swing45,
-                     'Swing90': self.swing90,
-                     'Swing135': self.swing135,
-                     'ROI Used (x, y, width, height)': self.ROI,
-                     'Instrument_Matrix': inst_mat},
-                    'Notes': notes,
-                    'Microscope Parameters': microscope_params
-                    }
+            metadata['Calibration'].update({
+                'Channel names': [f"State{i}" for i in range(5)],
+                'LC retardance': {f'LC{i}_{j}': np.around(getattr(self, f'lc{i.lower()}_{j}'), decimals=5)
+                                  for j in ['ext', '0', '45', '90', '135']
```

Comment:
Minor note: changing 'Ext' to 'ext' here will likely need to be updated in the calibration file reader as well. I'll add a commit shortly.